src/main/app/src/app/informatie-objecten/informatie-object-view/informatie-object-view.component.html

```diff
@@ -61,47 +61,49 @@
                             </div>
                         </mat-tab>
 
-                        <mat-tab id="historie_tab" label="{{'audittrail' | translate}}">
-                            <div class="table-wrapper audit-table">
-                                <table mat-table [dataSource]="auditTrail" matSort>
+                        <mat-tab id="historie_tab" label="{{'historie' | translate}}">
+                            <div class="table-wrapper historie-table">
```

Comment:
in less noem je de class historie en hier dus historie-table, nu mis je de scrollbar bij een lange historie